client/landing/gutenboarding/onboarding-block/design-selector/design-grid/index.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { Tooltip } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+import { isEnabled } from '@automattic/calypso-config';
+
+/**
+ * Internal dependencies
+ */
+import Badge from '../../../components/badge';
+import MShotsImage from '../../../components/mshots-image';
+import availableDesigns, {
+	getDesignImageUrl,
+	getDesignUrl,
+	mShotOptions,
+} from '../../../available-designs';
+import JetpackLogo from 'calypso/components/jetpack-logo'; // @TODO: extract to @automattic package
+import type { Design } from '../../../stores/onboard/types';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const makeOptionId = ( { slug }: Design ): string => `design-grid__option-name__${ slug }`;
+
+interface Props {
+	designs?: Design[];
+	isGridMinimal?: boolean;
+	locale: string;
+	onSelect: ( design: Design ) => void;
+}
+
+const DesignSelector: React.FunctionComponent< Props > = ( {
+	designs = ( availableDesigns?.featured as unknown ) as Design[],
+	isGridMinimal,
+	locale,
```

Comment:
Suggestion: switch to `useLocale` instead?

Reply:
When thinking of this `DesignGrid` component as a package (to be used outside of Gutenboarding app), we shouldn't assume there will be a i18n wrapper similar to the one in Gutenboarding to provide the locale. So, similar to how `launch`, `plans-grid` and `domain-picker` work, we should pass a `locale` prop to this one too. Actually, we should seriously consider to extract `DesignGrid` to a package since we're having this conversation 😁

Reply:
That's a good point — and definitely yes for extraction to package. It definitely won't hurt